src/main/scala/nl/biopet/bioconda/BiocondaPlugin.scala

```diff
@@ -0,0 +1,351 @@
+
+  override def requires: Plugins = {
+    GitPlugin && SbtGithubReleasePlugin
+  }
+
+  object autoImport extends BiocondaKeys
+
+  import autoImport._
+
+  override def projectSettings: Seq[Setting[_]] = Def.settings(
+    biocondaBranch := (normalizedName in Bioconda).value,
+    name in Bioconda := normalizedName.value,
+    biocondaCommand := (name in Bioconda).value,
+    biocondaMainGitUrl := "https://github.com/bioconda/bioconda-recipes.git",
+    biocondaMainBranch := "master",
+    biocondaUpdatedRepository := initBiocondaRepo.value,
+    biocondaUpdatedBranch := updateBranch().value,
+    biocondaRepository := new File(target.value, "bioconda"),
+    biocondaRecipeDir := new File(target.value, "recipes"),
+    biocondaBuildNumber := 0,
+    biocondaRequirements := Seq("openjdk"),
+    biocondaBuildRequirements := Seq(),
+    biocondaNotes := defaultNotes.value,
+    biocondaSummary := defaultSummary.value,
+    biocondaDefaultJavaOptions := Seq(),
+    biocondaCreateVersionRecipes := createRecipes(latest = false).value,
+    biocondaCreateLatestRecipe := createRecipes(versions = false).value,
+    biocondaCreateRecipes := createRecipes().value,
+    biocondaLicense := (licenses in Bioconda).value.toList.headOption
+      .getOrElse("No license", "")
+      ._1,
+    biocondaTestCommands := Seq(),
+    biocondaCommitMessage := s"Automated update for recipes of ${(name in Bioconda).value}",
+    biocondaAddRecipes := addRecipes().value,
+    biocondaTestRecipes := testRecipes.value,
+    biocondaOverwriteRecipes := false,
+    biocondaPushRecipes := pushRecipe.value,
+    biocondaPullRequestBody := defaultPullRequestBody.value,
+    biocondaPullRequestTitle := defaultPullRequestTitle.value,
+    biocondaPullRequest := createPullRequest.value,
+    biocondaRelease := release.value
+  )
+
+  override def globalSettings: Seq[Def.Setting[_]] = Def.settings(
+    )
+
+  private def release: Def.Initialize[Task[Unit]] =
+    Def
+      .task {}
+      .dependsOn(biocondaPullRequest)
+      .dependsOn(biocondaPushRecipes)
+      .dependsOn(biocondaTestRecipes)
+      .dependsOn(biocondaAddRecipes)
+      .dependsOn(biocondaCreateRecipes)
+
+  private def initBiocondaRepo: Def.Initialize[Task[File]] = {
+    Def.task {
+      val initialized: Boolean =
+        new File(biocondaRepository.value, ".git").exists()
+      val git = GitKeys.gitRunner.value
+      val s = streams.value
+      val local = biocondaRepository.value
+      val upstream = biocondaMainGitUrl.value
+      val origin = biocondaGitUrl.value
+      val branch = biocondaMainBranch.value
+
+      //Check if git repo already exits
+      if (!initialized) {
+        git.apply("init")(local, s.log)
+      }
+
+      // Check if remotes have been set properly
+      val remotes: Array[String] =
+        git.apply("remote")(local, s.log).split("\\n")
+
+      if (remotes.contains("upstream")) {
+        git.apply("remote", "set-url", "upstream", upstream)(local, s.log)
```

Comment:
with this you overwrite the current one, not sure if that is what you want without notice. First check if it's the same else an exception is maybe better.